### PR TITLE
[cds] Make old data migrations not applicable

### PR DESCRIPTION
**Pull request by alexey-klimuk**
Because we will switch to new cds update type

### CODE REVIEW

**Comment by alexey-klimuk** [comment]
@matthewford  could you please check and let me know if you agree with that.

File: db/data_migrations/20121204143748_add_hidden_commodities.rb [hunk 1]
```diff
@@ -19,6 +19,7 @@
   up do
     applicable {
       HiddenGoodsNomenclature.where(goods_nomenclature_item_id: CODES).count != CODES.size
```
Comment (matthewford) [inline db/data_migrations/20121204143748_add_hidden_commodities.rb:21]: We might still need hidden gonos I will need to check

**Review by matthewford — APPROVED** [review]

File: db/data_migrations/20121204143748_add_hidden_commodities.rb (hunk 1 above)
Comment (alexey-klimuk) [reply db/data_migrations/20121204143748_add_hidden_commodities.rb:21]: I'll import them together with chapters & notes vis sql dump

Comment (alexey-klimuk) [reply db/data_migrations/20121204143748_add_hidden_commodities.rb:21]: + measurement_unit_abbreviations

**Pull request merged**